src/installer.js

```diff
@@ -44,6 +44,14 @@ var spawn = function (options, command, args, callback) {
   })
 
   spawnedProcess.on('error', function (err) {
+    if (err.name = 'ENOENT') {
+      var installer = process.platform === 'darwin' ? 'brew' : 'apt-get'
```

Comment:
I know 😢 

@malept / @unindented any canonical sources you'd recommend for this, or would it be better to prepend an "e.g."?